hack.go

```diff
@@ -1,5 +1,3 @@
-// +build hack
```

Comment:
We really do want to have this tag here.

Reply:
What's the reason behind it? We had https://github.com/knative/test-infra/blob/master/scripts/dummy.go which doesn't have build tags.

Reply:
It was a mistake to not have build tags on the original file. We are using this file for go mod control, but no code should be here.

Reply:
Got it, I have added it back.